engine/jobmaster/dm/task_manager.go

```diff
@@ -287,7 +287,8 @@ func (tm *TaskManager) allFinished(ctx context.Context) bool {
 			if runningTask.Unit != frameModel.WorkerDMLoad {
 				return false
 			}
-		case dmconfig.ModeDump:
+		case dmconfig.ModeDump, dmconfig.ModeLoad, dmconfig.ModeLoadSync:
```

Comment:
```suggestion
		case dmconfig.ModeDump, dmconfig.ModeLoad:
```

sync unit will never finish

Reply:
Thank you for pointing this out. Will change this back.